test/extended/operators/recover.go

```diff
@@ -93,7 +94,129 @@ var _ = g.Describe("[sig-arch] Managed cluster should recover", func() {
 
 			healthy = nil
 			for _, operator := range currentOperators.Items {
-				if hasCondition(&operator, config.OperatorAvailable, config.ConditionTrue) && hasCondition(&operator, config.OperatorDegraded, config.ConditionFalse) {
+				if isHealthy(operator) {
+					healthy = append(healthy, operator)
+				}
+			}
+
+			if len(healthy) < len(operators.Items) {
+				return false, nil
+			}
+			if len(healthy) > len(operators.Items) {
+				return false, fmt.Errorf("found %s operators that were not in original set", strings.Join(operatorNames(healthy).Difference(operatorNames(operators.Items)).List(), ", "))
+			}
+			if !operatorNames(healthy).Equal(operatorNames(operators.Items)) {
+				return false, fmt.Errorf("found %s operators that were not in original set", strings.Join(operatorNames(healthy).Difference(operatorNames(operators.Items)).List(), ", "))
+			}
+			return true, nil
+		})
+		if err != nil {
+			buf := &bytes.Buffer{}
+			w := tabwriter.NewWriter(buf, 0, 1, 1, ' ', 0)
+			for _, operator := range currentOperators.Items {
+				fmt.Fprintf(w, "%s\t%s\t%s\n", operator.Name, conditionStatus(&operator, config.OperatorAvailable), conditionStatus(&operator, config.OperatorDegraded))
+			}
+			w.Flush()
+			o.Expect(err).ToNot(o.HaveOccurred(), fmt.Sprintf("Operators never became available: %s\n%s", strings.Join(operatorNames(operators.Items).Difference(operatorNames(healthy)).List(), ", "), buf.String()))
+		}
+	})
+	g.It("when operator-owned operand namespace is deleted [Disruptive]", func() {
```

Comment:
Tags should be at the top next to `g.Describe`, also add there one more unique tag so we can ensure we select only these in that separate test-suite we want to add. Something like `[OperatorRecovery]`, for example.

Reply:
I was following suit w/ the other test-case in this file: https://github.com/openshift/origin/blob/0fae074b297b0d7009fdd2cea2c37ec99295af16/test/extended/operators/recover.go#L28 - I'll move the [Disruptive] for _both_ above, in the `g.Describe` though.